index.bs

```diff
@@ -905,6 +905,49 @@ authorizing an authenticator with which to complete the operation.
 </div>
 
 
+
+### Inform user platform authenticator capability - isPlatformAuthenticatorReady() method ### {#isPlatformAuthenticatorReady}
+
+<div link-for-hint="WebAuthentication/isPlatformAuthenticatorReady">
+This method can be used by [=[RPS]=] to inform user they can create a new credential with one or more platform authenticators.
+The method gives [=[RPS]=] the ability to inform the client that they support registering credentials with <a>platform 
+authenticators</a> while giving users control over whether they want to share this information. When the 
+method is invoked, the <a>client</a> will search for the existence of an <a>platform authenticator</a> capable of 
+registering credentials. Upon finding one, the <a>client</a> will determine whether the user would be interested in 
+creating a credential based on a number of factors, which may include the user's explicitly stated intention. If such interest
+is found, the promise will be resolved with the value of <i>true</i>. Based on the result, the [RPS] can take further actions
+to guide the user to create a credential. 
+
+This method doesn't accept any argument and returns a boolean object as the response. 
+
+<pre class="idl">
+    [SecureContext]
+    partial interface PublicKeyCredential {
+        [Unscopable] Promise < boolean > isPlatformAuthenticatorReady();
+    };
+</pre>
+
+When this method is invoked, the user agent MUST execute the following algorithm: 
+
+1. Let |promise| be a [=a new Promise=]. Return |promise| and execute the following steps [=in parallel=].
+
+1. Enumerate all the [=platform authenticators=] on this platform. If there are no such authenticators, resolve |promise| with
+    a value of false and terminate this algorithm.
```

Comment:
This should probably just not resolve, not resolve in a way leaking information without user consent.